components/ciphertool/src/main/java/org/wso2/ciphertool/CipherTool.java

```diff
@@ -383,7 +383,7 @@ private static void updateDeploymentConfigurationWithEncryptedKeys(Map<String, S
                 boolean found = false;
                 for (String line : lines) {
                     if (found) {
-                        if (line.matches("[.+]")) {
+                        if (line.matches("\\[.+\\]")) {

```

Comment:
Was it only removing configurations after the secrets section?

Reply:
yes.